app/decorators/generic_object_definition_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class GenericObjectDefinitionDecorator < MiqDecorator
+  def self.fonticon
+    'fa fa-file-text-o'
+  end
+
+  def self.fileicon
+    'svg/generic-object.svg'
```

Comment:
@dclarizio @AparnaKarve This is wrong. There should not be 2 fonticon definitions and it shouldn't call an SVG if there were.